micov.yml

```diff
@@ -0,0 +1,131 @@
+name: micov_test
+channels:
+  - conda-forge
+dependencies:
+  - aws-c-auth=0.8.1=h6661f4c_0
```

Comment:
This environment pins to many packages which we don't need to worry about. It also pins us to very specific versions of packages we do care about, but where I'm unaware of a reason to do so. 

Can the versions and pins from `setup.py` be reflected here instead?